### PR TITLE
Increase process group prefix length

### PR DESCRIPTION
**Pull request by johscheuer**
# Description

Increase the length of the prefix.

## Type of change

*Please select one of the options below.*

- Bug fix (non-breaking change which fixes an issue)

# Discussion

-

# Testing

local

# Documentation

-

# Follow-up

-

### CODE REVIEW

File: api/v1beta1/foundationdbcluster_types.go [hunk 1]
```diff
@@ -194,7 +194,7 @@ type FoundationDBClusterSpec struct {
 	// This must be a valid Kubernetes label value. See
 	// https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set
 	// for more details on that.
-	// +kubebuilder:validation:MaxLength=32
+	// +kubebuilder:validation:MaxLength=64
```
Comment (johscheuer) [inline api/v1beta1/foundationdbcluster_types.go:197]: TODO: Remove the link above. The prefix will be used a s value not as key.

Comment (johscheuer) [reply api/v1beta1/foundationdbcluster_types.go:197]: Another note we should not exceed 63 (this is the maximum for a label value). I would guess that `cluster-controller` (19 chars) is currently the longest role so we should be fine to have 63-20= 43 as `MaxLength`

**Review by brownleej — CHANGES_REQUESTED** [review]
Requesting changes to address the previous discussion on the length limitation.

File: api/v1beta1/foundationdbcluster_types.go [hunk 2]
```diff
@@ -194,7 +194,7 @@ type FoundationDBClusterSpec struct {
 	// This must be a valid Kubernetes label value. See
 	// https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set
 	// for more details on that.
-	// +kubebuilder:validation:MaxLength=32
+	// +kubebuilder:validation:MaxLength=43
```
[the remark this answers is not in the document]
Comment (johscheuer) [reply api/v1beta1/foundationdbcluster_types.go:197]: I think that makes sense but not as part of this PR. I can open an issue and we can look at this in another PR.

[the remark this answers is not in the document]
Comment (johscheuer) [reply api/v1beta1/foundationdbcluster_types.go:197]: Opened: https://github.com/FoundationDB/fdb-kubernetes-operator/issues/1096

**Review by brownleej — APPROVED** [review]
LGTM. I'm guessing that the deployment YAML change is going to get wiped out by a rebase. Can you rebase to make sure everything is consistent?

**Comment by johscheuer** [comment]
The files checks have passed so I'm going ahead and merge this PR.

**Pull request merged**